src/Java.Interop/Properties/AssemblyInfo.cs

```diff
@@ -28,3 +28,10 @@
 	"814f144e5d817efc4c6502cc012df310783348304e3ae38573c6d658c234025821fda87a0be8a0" +
 	"d504df564e2c93b2b878925f42503e9d54dfef9f9586d9e6f38a305769587b1de01f6c0410328b" +
 	"2c9733db")]
+[assembly: InternalsVisibleTo (
```

Comment:
I'm torn between accepting this, or just copying the `JniLocationException` type into `Java.Runtime.Environment.dll` to avoid the `InternalsVisibleTo`… :-/